cmd/azure/azure.go

```diff
@@ -256,10 +167,28 @@ func run() error {
 		}
 	}
 
-	// wait for the cluster to come up
-	//	err = waitForBootstrapComplete(ctx, config, rootOpts.dir)
-	//	err = destroybootstrap.Destroy(rootOpts.dir)
-	//	err = waitForInstallComplete(ctx, config, rootOpts.dir)
 
+	// waiting routine
+	config, err := clientcmd.BuildConfigFromFlags("", filepath.Join(ec.Directory, "auth", "kubeconfig"))
+				if err != nil {
+					logrus.Fatal(errors.Wrap(err, "loading kubeconfig"))
+				}
+
+	// TODO: Implement context
+	ctx:= context.Background()
```

Comment:
I'm surprised gofmt isn't barfing on this... looks like CI maybe isn't running because this is being merged into the dev branch?

Reply:
yes. need to sort this fast!